Libraries/LibWebView/Utilities.cpp

```diff
@@ -112,7 +112,7 @@ ErrorOr<Vector<ByteString>> get_paths_for_helper_process(StringView process_name
     auto application_path = TRY(application_directory());
     Vector<ByteString> paths;
 
-#if !defined(AK_OS_MACOS) && !defined(AK_OS_WINDOWS)
+#if !defined(AK_OS_MACOS)
     auto prefix = find_prefix(LexicalPath(application_path));
     TRY(paths.try_append(LexicalPath::join(prefix.string(), libexec_path, process_name).string()));
```

Comment:
surely libexec isn't a thing on windows?

Shouldn't the helpers be in the same directory as the Ladybird.exe file? Is application_directory() returning one up from where it should?